SafeSnap-Agents/Cooldown-Alert/src/agent.ts

```diff
@@ -0,0 +1,95 @@
+import { 
+  BlockEvent,
+  Finding, 
+  HandleBlock,
+  HandleTransaction,
+  FindingSeverity, 
+  FindingType,
+  getEthersProvider,
+  TransactionEvent,
+} from "forta-agent";
+import {
+  propQuestionCreateAbi,
+} from "./abi";
+import DataFetcher from "./data.fetcher";
+
+const REALITIO_ERC20: string = "0x8f1CC53bf34932591177CDA24723486205CA7510".toLowerCase();
```

Comment:
This address you are using is not the oracle of the module you have, you need to fetch the oracle from the Module contract, because the only address we can have in the agent is the address of the module